.changeset/rotten-panthers-share.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-common': minor
+'@backstage/plugin-catalog-backend-module-aws': minor
```

Comment:
Both of these can be `patch`, since there are no user facing changes (and these are 0.x version packages).